modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,534 @@
+
+Ensure that the advertised hostnames are operating on the correct network interfaces.
+
+Example:
+```bash
+grep -A1 advertised /etc/redpanda/redpanda.yaml
+    advertised_kafka_api:
+    -   address: 54.244.110.128
+--
+    advertised_rpc_api:
+        address: 172.31.40.237
+```
+
+[[continuous-db]]
+=== Continuous data balancing is configured
+
+Ensure that xref:manage:cluster-maintenance/continuous-data-balancing.adoc[Continuous Data Balancing] is configured and enabled.
+
+Example: 
+```bash
+rpk cluster config get partition_autobalancing_mode
+```
+
+Output:
+```bash
+continuous
+```
+
+[[debug-bundle]]
+=== Generate a debug bundle
+
+Ensure that you’re able to generate a debug bundle from each host and upload to Redpanda Support.
+
+```bash
+sudo rpk debug bundle
+Creating bundle file...
+Debug bundle saved to '1711372017-bundle.zip'
+```
+
+[[topic-rf]]
+=== Topic replication factor
+
+Ensure that the topics replication factor are greater than one.
+
+Example:
+```bash
+rpk topic list
+```
+
+Output:
+```bash
+NAME   PARTITIONS  REPLICAS
+bad    1           1
+good   1           3
+```
+
+[[maintenance-mode]]
+=== Brokers in maintenance mode
+
+Ensure that no brokers are in maintenance mode.
+
+Example:
+
+```bash
+rpk cluster maintenance status
+```
+
+Output:
+
+```bash
+NODE-ID  ENABLED  FINISHED  ERRORS  PARTITIONS  ELIGIBLE  TRANSFERRING  FAILED
+1        false    -         -       -           -         -             -
+2        false    -         -       -           -         -             -
+3        false    -         -       -           -         -             -
+```
+
+[[decom-state]]
+== Brokers in a decommissioned state
+
+Ensure that no brokers are in a decommissioned state.
+
+Example:
+```bash 
+rpk redpanda admin brokers list
+```
+
+Output:
+```bash
+NODE-ID  NUM-CORES  MEMBERSHIP-STATUS  IS-ALIVE  BROKER-VERSION
+0        1          active             true      v23.2.25 - 8530f7c17346eb7c7459c3912f012299f3762efd
+1        1          active             true      v23.2.25 - 8530f7c17346eb7c7459c3912f012299f3762efd
+2        1          active             true      v23.2.25 - 8530f7c17346eb7c7459c3912f012299f3762efd
+```
+
+
+== Level 2 Production Readiness
+
+At level 2 we are looking to ensure that you are best able to monitor and support your environment on a sustained basis. The following is a list of recommendations to help this.
+
+. Do you have a development or test environment configured where you can evaluate upgrades and new versions before rolling them straight to production.
+
+. Monitoring: Prometheus and Grafana, and/or Datadog or equivalent are configured to scrape metrics from all Redpanda brokers at a regular interval.
+
+. System logs are being captured and stored for an appropriate period of time (usually at least 7 days). On bare metal, this may be journald however on Kubernetes you may need to have fluentd or equivalent configured and logs sent to a central location.
```

Comment:
```suggestion
- System logs are being captured and stored for an appropriate period of time (usually at least 7 days). On bare metal, this may be journald, however on Kubernetes you may need to have fluentd or equivalent configured and logs sent to a central location.
```

Reply:
See also: ...could not find any specific guidance on capturing system logs. @tmgstevens , @hcoyote Where do you typically point customers when you tell them to capture system logs? Seems odd that it's not currently documented. Content gap?